projects/ff4j-dashboards/src/app/dashboard.browser.module.ts

```diff
@@ -13,7 +13,7 @@ import { RouterModule } from '@angular/router';
 export class DashboardBrowserModule {
   constructor(private injector: Injector) {}
 
-  ngDoBootstrap() {
+  ngDoBootstrap(): any {
```

Comment:
should not the return type of this function be **void** instead of **any**?

Reply:
I couldnt find what is the appropriate one . Any thoughts ? Maybe we can change it later